base/install_opcua.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -ex
+
+. /opt/epics/install-functions.sh
+
+lnls-get-n-unpack -l https://github.com/epics-modules/opcua/releases/download/v${OPCUA_VERSION}/IOC_opcua-${OPCUA_VERSION}_Base-${EPICS_BASE_VERSION}_debian${DEBIAN_VERSION%.*}.tar.gz
+mv binaryOpcuaIoc opcua
+
+#install_module opcua OPCUA "
+#EPICS_BASE=${EPICS_BASE_PATH}
+#"
+
+module_releases="
+OPCUA=${EPICS_MODULES_PATH}/opcua
+"
+echo "$module_releases" >> ${EPICS_RELEASE_FILE}
+
+cd opcua
+
+echo "
+EPICS_BASE=${EPICS_BASE_PATH}
+" > configure/RELEASE.local
+
+make -j${JOBS}
+
+EPICS_HOST_ARCH=$(${EPICS_BASE_PATH}/startup/EpicsHostArch)
```

Comment:
`startup/EpicsHostArch` will be removed from [future releases of base][1]. You should call the Perl script directly.

```bash
perl ${EPICS_BASE_PATH}/lib/perl/EpicsHostArch
```

[1]: https://github.com/epics-base/epics-base/blame/cb1571783b4793cb9c545e5736659ee5ed7d03bb/startup/EpicsHostArch#L10